3rdparty/openvx/include/openvx_hal.hpp

```diff
@@ -462,7 +493,11 @@ inline int ovx_hal_warpAffine(int atype, const uchar *a, size_t astep, int aw, i
         {
         case CV_HAL_BORDER_CONSTANT:
             border.mode = VX_BORDER_CONSTANT;
+#if VX_VERSION > VX_VERSION_1_0
```

Comment:
this is absolutely the same code repeated in several places; can we add a helper inline function to set the constant border value?